qiskit/transpiler/passes/layout/vf2_utils.py

```diff
@@ -141,6 +142,15 @@ def score_layout(
     )
 
 
+def build_dummy_target(coupling_map) -> Target:
```

Comment:
I wanted to suggest using `FakeTarget` instead of `build_dummy_target`, but unfortunately it's not applicable here because `FakeTarget` should only exist in python space and this target is given to the rust path.

Reply:
yeah, I didn't test it, but I think `_FakeTarget` would cause Rust space to see a target that has a defined number of qubits by no operations at all